src/tactic/lint/type_classes.lean

```diff
@@ -105,8 +105,9 @@ If you don't know what priority to choose, use priority 100.
 See note [lower instance priority] for instructions to change the priority.",
   auto_decls := tt }
 
-/-- Reports declarations of types that do not have an associated `inhabited` instance. -/
-private meta def has_inhabited_instance (d : declaration) : tactic (option string) := do
+/-- Reports declarations of types that do not have an associated `nonempty` instance.
+It suffices to have an `inhabited` or `unique` instance rather than a `nonempty` instance. -/
```

Comment:
Let's make this suggestion a bit stronger.
```suggestion
If possible, we prefer a computable `inhabited` or `unique` instance rather than a `nonempty` instance. -/
```